build-n-publish.sh

```diff
@@ -3,64 +3,60 @@
 pkgName="doppler-webapp"
 pkgVersion=${1:-"v0.0.0-build0"}
 cdnBaseUrl=${2:-"//cdn.fromdoppler.com/$pkgName"}
-pkgVersionQa="qa-$pkgVersion"
-pkgVersionInt="int-$pkgVersion"
-pkgVersionDevelopment="dev-$pkgVersion"
 pkgBuild=${4:-0}
 pkgCommitId=${5:-$(git rev-parse HEAD)}
+environments="int qa development production"
 
 # Exit immediately if a command exits with a non-zero status.
 set -e
 
+# Stop script if unbound variable found (use ${var:-} if intentional)
+set -u
+
 # Lines added to get the script running in the script path shell context
 # reference: http://www.ostricher.com/2014/10/the-right-way-to-get-the-directory-of-a-bash-script/
 cd $(dirname $0)
 
-sh ./verify-w-docker.sh
-
-# Force pull the latest image version due to the cache not always is pruned immediately after an update is uploaded to docker hub
-docker pull dopplerrelay/doppler-relay-akamai-publish
+# To avoid issues with MINGW y Git Bash, see:
+# https://github.com/docker/toolbox/issues/673
+# https://gist.github.com/borekb/cb1536a3685ca6fc0ad9a028e6a959e3
+export MSYS_NO_PATHCONV=1
+export MSYS2_ARG_CONV_EXCL="*"
 
-sh ./build-w-docker.sh $pkgVersion $cdnBaseUrl production $pkgVersion $pkgBuild $pkgCommitId
-docker run --rm \
-    -e AKAMAI_CDN_HOSTNAME \
-    -e AKAMAI_CDN_USERNAME \
-    -e AKAMAI_CDN_PASSWORD \
-    -e AKAMAI_CDN_CPCODE \
-    -e "PROJECT_NAME=$pkgName" \
-    -e "VERSION_NAME=$pkgVersion" \
-    -v `pwd`/build:/source \
-    dopplerrelay/doppler-relay-akamai-publish
+echo Publishing to Docker and Akamai...
+echo pkgName: $pkgName
+echo cdnBaseUrl: $cdnBaseUrl
+echo pkgVersion: $pkgVersion
+echo pkgBuild: $pkgBuild
+echo pkgCommitId: $pkgCommitId
 
-sh ./build-w-docker.sh $pkgVersion $cdnBaseUrl qa $pkgVersion $pkgBuild $pkgCommitId
-docker run --rm \
-    -e AKAMAI_CDN_HOSTNAME \
-    -e AKAMAI_CDN_USERNAME \
-    -e AKAMAI_CDN_PASSWORD \
-    -e AKAMAI_CDN_CPCODE \
-    -e "PROJECT_NAME=$pkgName" \
-    -e "VERSION_NAME=$pkgVersionQa" \
-    -v `pwd`/build:/source \
-    dopplerrelay/doppler-relay-akamai-publish
+# TODO: It could break concurrent deployments with different docker accounts
+docker login -u="$DOCKER_WEBAPP_USERNAME" -p="$DOCKER_WEBAPP_PASSWORD"
```

Comment:
I think that it is not necessary because we do not upload anything to Docker Hub.